reference/test/solver/bicg_kernels.cpp

```diff
@@ -89,6 +89,16 @@ class Bicg : public ::testing::Test {
                           .with_reduction_factor(r<value_type>::value)
                           .on(exec))
                   .on(exec)),
+          bicg_factory_big2(
+              Solver::build()
+                  .with_criteria(
+                      gko::stop::Iteration::build().with_max_iters(100u).on(
+                          exec),
+                      gko::stop::ImplicitResidualNormReduction<
```

Comment:
Wow, that is awful formatting. It's tolerable, I guess.

Reply:
Yeah, sometimes clang-format cannot do too much, but it tries to keep atleast the namespaces and the object name together.